### PR TITLE
No large-portrait or large-landscape

### PR DESCRIPTION
**Pull request by i-like-robots**
This removes the `--large-portrait` and `--large-landscape` variants. They were initially created so that the `--large` teaser may be forced into a fixed orientation instead of responding.

These variations are not used on FT.com and have not been since November 2016.

- https://github.com/search?q=org%3AFinancial-Times+large-landscape&type=Code
- https://github.com/search?q=org%3AFinancial-Times+large-portrait&type=Code

CC #101

### CODE REVIEW

**Comment by i-like-robots** [comment]
Oh - I'm also having discussions with design about the behaviour of the `large` teaser  being different at the medium breakpoint. Depending on the result of this I may have another PR.

File: origami.json [hunk 1]
```diff
@@ -324,13 +324,13 @@
 			}
 		},
 		{
-			"title": "Hero Extra",
+			"title": "Hero Extra Centered",
```
Comment (gvonkoss) [inline origami.json]: Ugh I can't comment on the lines above, I realise this isn't your doing, but it says Hero Centred 😞 could you change that too, pls 😬

**Review by gvonkoss — APPROVED** [review]

File: origami.json (hunk 1 above)
Comment (i-like-robots) [reply origami.json]: Would you prefer British or American spelling? The modifier class name is `o-teaser--centre` so "centred" everywhere?

Comment (gvonkoss) [reply origami.json]: You know, I'm not sure, we use American for 'colors' in our docs. Let's go with British for now and I'll make a note of updating that documentation when I find it. 🙏

**Pull request merged**